docs/source/index.md

```diff
@@ -1,21 +1,99 @@
-# Welcome to MALA's documentation!
+![image](./img/logos/mala_horizontal.png)
+
+# Welcome to MALA!
+
+## What is MALA?
+
+MALA is a software package for building ML models that replace
+density functional theory (DFT) calculations. DFT is **the** most
+popular method for simulating materials on a quantum level and predicting 
+materials properties, and employed by researcher throughout the world. 
+
+While DFT is very efficient compared to other quantum mechanical simulation
+techniques, it still scales cubically with number of atoms and temperature 
+simulated, making large scale simulation at arbitrary temperatures very 
+computationally challenging. 
+
```

Comment:
Although DFT is more efficient compared to other quantum mechanical simulation techniques, it still exhibits cubic scaling with the number of atoms. This limitation poses significant computational challenges for conducting large-scale simulations.